.travis.yml

```diff
@@ -14,11 +14,11 @@ before_install:
   - echo "https://$GH_TOKEN:@github.com" > .git/credentials
   - gem install asciidoctor
 install:
-- ./mvnw install -P docs -q -U -DskipTests=true -Dmaven.test.redirectTestOutputToFile=true
+- ./mvnw install -P docs -q -U -DskipTests=true -Dmaven.test.redirectTestOutputToFile=false
 - ./docs/src/main/asciidoc/ghpages.sh
 script:
-- '[ "${TRAVIS_PULL_REQUEST}" != "false" ] || ./mvnw --settings .settings.xml deploy -P release -nsu -Dmaven.test.redirectTestOutputToFile=true'
-- '[ "${TRAVIS_PULL_REQUEST}" = "false" ] || ./mvnw install -nsu -Dmaven.test.redirectTestOutputToFile=true'
+- '[ "${TRAVIS_PULL_REQUEST}" != "false" ] || ./mvnw --settings .settings.xml deploy -P release -nsu -Dmaven.test.redirectTestOutputToFile=false'
```

Comment:
I think this PR is about changing true->false, but https://github.com/spring-cloud/spring-cloud-stream/commit/397ef6f5571b877ae5a4b12e6f1a55bf66d57677#diff-354f30a63fb0907d4ad57269548329e3 was done to change the maven target not to publish (as we have a bamboo job for that....)   So once we get the true/false value cleared up, I can make the appropriate changes.